### DISCUSSION TITLE
How i can read the Laravel code?

### DISCUSSION THREAD

**Discussion by diegogit03** [laravel/framework#50859] [Q&A]
Where i can start, i already read the Architecture concepts.

**Comment by janzcio** [comment]
It seems to me, thats a hex color, try it here 
https://mycolor.space/?hex=%23508598&sub=1

**Reply by diegogit03** [reply]
What?

**Reply by henzeb** [reply]
He's referring to the hashtag after the title of the post. He is mistaken. Probably because you haven't specified clearly what code...
